src/Microsoft.ML/CSharpApi.cs

```diff
@@ -9689,6 +9701,128 @@ public StochasticGradientDescentBinaryClassifierPipelineStep(Output output)
         }
     }
 
+    namespace Trainers
+    {
+
+        /// <summary>
+        /// Train a symbolic SGD.
+        /// </summary>
```

Comment:
NOt sure how much documentation we have about SymSGD. If any, might be nice to include it.

Reply:
Done.